### ISSUE TITLE
extra/Ssh start faild

### ISSUE THREAD

**Issue by Saikatsaha1996** [meefik/linuxdeploy#1325]
Can't start with ssh

**Comment by Saikatsaha1996** [comment]
And also want more help.. how to enable binfmt_misc from kernel? I mean which option in Kernel?

**Comment by Saikatsaha1996** [comment]
@meefik can you help me?..
![IMG_20220801_103350](https://user-images.githubusercontent.com/72664192/182076091-c961aab0-b03a-43b3-9a75-ceb443f5eb6c.jpg)

**Comment by Saikatsaha1996** [comment]
![IMG_20220801_104237](https://user-images.githubusercontent.com/72664192/182076867-3ee553c8-b460-4aae-91b4-895c29b3dbf5.jpg)

**Comment by Saikatsaha1996** [comment]
@meefik now maintenar not available? Is project is no longer available ?

**Comment by RyanEwen** [comment]
SSH worked once for me but the container I was using ran out of space (stored in 2gb file) so I created another using directory storage. SSH has never worked again. Tried creating profiles and containers from other OSes and cannot figure out why SSH no longer starts. Rebooted etc as well.

**Comment by RyanEwen** [comment]
I seem to have it working again by choosing /data/linux as the directory for my container. Maybe it just doesn't like ${EXTERNAL_DATA}/linux?

**Comment by Saikatsaha1996** [comment]
> I seem to have it working again by choosing /data/linux as the directory for my container. Maybe it just doesn't like ${EXTERNAL_DATA}/linux?

Yes my img store in ${EXTERNAL_DATA}/linux

**Comment by Saikatsaha1996** [comment]
> I seem to have it working again by choosing /data/linux as the directory for my container. Maybe it just doesn't like ${EXTERNAL_DATA}/linux?

![IMG_20220819_200106](https://user-images.githubusercontent.com/72664192/185641669-299e9362-fa4b-480a-a48f-80f76267ff42.jpg)

**Comment by Saikatsaha1996** [comment]
Yes i found problem when i installed another architecture Linux it was failed to start ssh vnc && all

**Comment by RyanEwen** [comment]
I don't think the issue with `{$EXTERNAL_DATA}` affects File storage (.img). Just Directory.

**Comment by Saikatsaha1996** [comment]
> Yes i found problem when i installed another architecture Linux it was failed to start ssh vnc && all

No problem is with another architecture..

When i installed amd64 debootstrap getting ssh error

**Comment by bortan1** [comment]
Has anyone managed to fix it?

**Comment by simmorsal** [comment]
> Has anyone managed to fix it?

The solution I chose was installing Alpine which worked without an issue
